packages/fuels/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fuels",
-  "version": "0.99.0",
+  "version": "0.99.1",
```

Comment:
Don't merge until https://github.com/FuelLabs/fuels-ts/pull/3602#discussion_r1935296050 is handled.

cc @luizstacio